app.jsx

```diff
@@ -11,69 +11,75 @@ import styled from "styled-components";
 
 import Colors from "./constants/colors";
 import Spacing from "./constants/spacing";
+import { BASE_STYLES, TAG_STYLES } from "./constants/styles";
 
-const parseNodes = (nodes, baseStyle = "normal") => {
+const getInlineStyle = (inlineStyle) => {
+    const isItalic = !!inlineStyle.match(/italic/);
+    const isBold = !!inlineStyle.match(/weight: 600|700|bold/);
+
+    let style = [];
+    if (isItalic) {
+        style = [...style, BASE_STYLES.ITALIC];
+    }
+    if (isBold) {
+        style = [...style, BASE_STYLES.BOLD];
+    }
+    
+    return style.length > 0 ? style : [];
+}
+
+const getStyle = ({name, attribs}) => {
+    const tagStyles = (name && TAG_STYLES[name] && [TAG_STYLES[name]]) || [];
```

Comment:
6. Whilst we could go with conditions (such as` if (name === "b") do x`) to ascertain the style of an element based on its name attribute, going through a lot of conditions can be costly in terms of performance, so my preference would be to store each option in an object that we access through the name key.